ArcadiaTeamsBot.CQRS/Handlers/GetServiceDeskRequestTypesHandler.cs

```diff
@@ -0,0 +1,40 @@
+﻿using ArcadiaTeamsBot.CQRS.Queries;
+using MediatR;
+using ServiceDesk.Abstractions.DTOs;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace ArcadiaTeamsBot.CQRS.Handlers
+{
+    class GetServiceDeskRequestTypesHandler : IRequestHandler<GetServiceDeskRequestTypesQuery, IEnumerable<ServiceDeskRequestTypeDTO>>
+    {
+        private const string requestTypesUrl = "https://arcadia-servicedesk-dev31.azurewebsites.net/api/intra/requestTypes";
+
+        public GetServiceDeskRequestTypesHandler(IHttpClientFactory clientFactory)
+        {
+            ClientFactory = clientFactory;
+        }
+
+        public IHttpClientFactory ClientFactory { get; }
+
+        public async Task<IEnumerable<ServiceDeskRequestTypeDTO>> Handle(GetServiceDeskRequestTypesQuery request, CancellationToken cancellationToken)
+        {
+            var httpRequest = new HttpRequestMessage(HttpMethod.Get, requestTypesUrl);
```

Comment:
To not do these all every time you want to get or post something to Service Desk I said you to create a service desk client class which will implement all possible interactions with Service Desk API.

Reply:
Service Desk client has been implemented